### DISCUSSION TITLE
How to load url from localhost

### DISCUSSION THREAD

**Discussion by hpposch** [cefsharp/CefSharp#3810] [General]
Hi all!
I am currently trying to load an url that points to a local webserver, but no matter what settings i apply, i can not get it working.
When i use a public url, everything works fine.
Only localhost urls are not loaded by the browser.
What am i missing?
Is there an example for that use case?

Thanks in advance for your help!

**Comment by amaitland** [comment]
> Only localhost urls are not loaded by the browser.

Does the localhost url load correctly in Chrome? Check the log file for errors.

**Reply by hpposch** [reply]
Yes when i use chrome it loads correctly

**Reply by amaitland** [reply]
Check the log file.

**Reply by hpposch** [reply]
Hmm..
I just did a complete rebuild and now it seems to load the page xD
I am really sorry for that, but i do not know why it is working now :)
Sorry for that!
